gammapy/datasets/spectrum.py

```diff
@@ -213,6 +213,12 @@ class SpectrumDataset(PlotMixin, MapDataset):
     stat_type = "cash"
     tag = "SpectrumDataset"
 
+    @property
+    def energy_range(self):
```

Comment:
I think this could be useful for `MapDataset` as well. If we rename e.g. to `energy_range_quantity` (or similar), then it would return `np.min(energy_min_map.quantity)` and `np.max(energy_min_map.quantity)`.